jolt-core/src/jolt/vm/pc.rs

```diff
@@ -138,23 +154,23 @@ pub struct PCPolys<F: PrimeField> {
 
 impl<F: PrimeField> PCPolys<F> {
   // TODO(sragss): precommit PC strategy
-  pub fn new_program(program: Vec<ELFRow>, trace: Vec<ELFRow>) -> Self {
-	assert!(is_power_of_two(program.len()));
-	assert!(is_power_of_two(trace.len()));
+  pub fn new_program(mut program: Vec<ELFRow>, mut trace: Vec<ELFRow>) -> Self {
+    Self::validate_program(&program, &trace);
```

Comment:
maybe we should put this behind `#[cfg(test)]`?

Reply:
We could – I was thinking the cost was fairly minimal compared to our other activities